Source/Facebook/FacebookClient.cs

```diff
@@ -602,7 +602,10 @@ private HttpHelper PrepareRequest(HttpMethod httpMethod, string path, object par
                     throw new ArgumentOutOfRangeException("httpMethod");
             }
 
-            request.ContentType = contentType;
+            if (contentType != null)

```

Comment:
can you use `string.IsNullOrEmpty` and also remove the braces like for etag below.